### PR TITLE
FIX mongo driver legacy-1.0.2 -> legacy-1.0.7

### PR DESCRIPTION
**Pull request by fgalan**
Upgrading driver hasn't impact in the code (it has impact in the system that builts the RPMs, i.e. jenkins, that will be upgraded after this PR got merged) but this PR includes some changes in the `LmtMongo` traces that were used the bug that motivates this upgrade and that can be useful in the future.

@kzangeli 
@crbrox

### CODE REVIEW

**Comment by fgalan** [comment]
(The release name mentions 1.0.6, because that was the original ideal. However, at the end the target is 1.0.7. Sorry...)

**Comment by kzangeli** [comment]
Only traces changed in the source code, right?

**Comment by kzangeli** [comment]
LGTM

**Comment by fgalan** [comment]
> Only traces changed in the source code, right?

Correct

File: src/lib/mongoBackend/MongoGlobal.cpp [hunk 1]
```diff
@@ -229,7 +229,7 @@ void releaseMongoConnection(DBClientBase* connection, std::auto_ptr<DBClientCurs
 #else
 #if MONGO_DRIVER_LEGACY_1_0_6_OR_NEWER
```
Comment (LeandroGuillen) [inline src/lib/mongoBackend/MongoGlobal.cpp]: Small typo: should be 1_0_7, right?

Comment (fgalan) [inline src/lib/mongoBackend/MongoGlobal.cpp]: Not really... that (1.0.6) was the driver version that introduces kill(). However, at the end we are not going to use kill() (details in #1568).

Anyway, this part of the code needs a little refactoring. Coming in a next commit.

Comment (fgalan) [inline src/lib/mongoBackend/MongoGlobal.cpp]: Fixed in 50cc10e

**Comment by LeandroGuillen** [comment]
LGTM

**Comment by fgalan** [comment]
Sucessfull test by @jcanonav in INT environment. Ready to merge.

**Pull request merged**